crates/bls-gadgets/src/hash_to_group.rs

```diff
@@ -161,13 +161,12 @@ impl HashToGroupGadget<Bls12_377_Parameters, Bls12_377_Fq> {
             )?;
 
         let mut crh_bits = pedersen_hash.x.to_bits_le().unwrap();
-        crh_bits.reverse();
         // The hash must be front-padded to the nearest multiple of 8 for the LE encoding
         loop {
             if crh_bits.len() % 8 == 0 {
                 break;
             }
-            crh_bits.insert(0, Boolean::constant(false));
```

Comment:
This is going to insert the element at the opposite end of the vector. Are we sure this is okay?

Reply:
I believe so, because a few lines up we removed the reverse